config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
   root 'pages#index'
-  get '/signup',  to: 'users#new'
-  post '/signup', to: 'users#create'
-  resources :users
+  get    '/signup',  to: 'users#new'
+  get    '/login',   to: 'sessions#new'
+  delete '/logout',  to: 'sessions#destroy'
+
+  resources :users, only: [:create] do
+    collection do
+      get :profile
+    end
+  end
+
+  resources :sessions, only: [:create, :destroy]
```

Comment:
I think we only need `create` action here because we have already defined a delete request below

Reply:
fixed from
```
resources :sessions, only: [:create, :destroy]
```
to
```
resources :sessions, only: [:create]
```